nixos/modules/services/monitoring/glpi-agent.nix

```diff
@@ -0,0 +1,91 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.glpiAgent;
+
+  settingsType =
+    with lib.types;
+    attrsOf (oneOf [
+      bool
+      int
+      str
+      (listOf str)
+    ]);
+
+  formatValue =
+    v:
+    if lib.isBool v then
+      (if v then "1" else "0")
+    else if lib.isList v then
+      lib.concatStringsSep "," (map toString v)
```

Comment:
`toString` shouldn't be necessary here because the type check will only allow strings to begin with.